### DISCUSSION TITLE
[keyball39]ジャンパに失敗しランドが剥がれた結果、OLEDが点灯しない

### DISCUSSION THREAD

**Discussion by Shuyawa89** [Yowkees/keyball#659] [Q&A] [answered]
お世話になっております。
先日、Keyball39を購入し、組み立てを進めております。
自作キーボード初心者であるため、組み立て中に次のような問題が発生し、ご相談させていただきたくご連絡いたしました。

## 問題点
右手側（トラックボールあり）のOLED部のジャンパ作業に失敗した影響か、右手側のOLEDが点灯しない状況です。
そこで、どの箇所を銅線などでジャンパすべきか、具体的な箇所をご教授いただけますと幸いです。
また、画像から他にの原因等が見つかる場合にも教えていただきたいです。

## 現状
キーボード自体の動作は正常で、左手側・右手側どちらにケーブルを接続してもファームウェアの書き込みは問題なく行え、両手の全てのキーが動作することを確認しております。
そのため、現時点では動作に支障はないと判断し、組み立てを進めましたが、OLEDの点灯のみが問題となっています。
左手側のOLEDは正常に点灯しており、ジャンパに失敗した右手側のみが点灯しないといった状況です。

参考として、[Keyball61の場合の修理例](https://github.com/Yowkees/keyball/discussions/277#discussioncomment-6501523)を確認しましたが、Keyball39において同様の方法が適用できるか不明なため、こちらで確認させていただきました。

お忙しいところ恐れ入りますが、お手隙の際にご回答いただければ幸いです。
どうぞよろしくお願いいたします。

### OLEDが点灯しない右手側の様子

| 場所 | 画像 |
|--------|--------|
| ジャンパに失敗した箇所 | ![IMG_4302](https://github.com/user-attachments/assets/c249fbc5-407c-4694-942d-063b110344d0) |
| ジャンパに失敗した箇所 別角度 | ![IMG_4318](https://github.com/user-attachments/assets/303ba6ab-3954-4daa-94d1-68de89f0e1dd) | 
| 表面 OLED部 |![IMG_4316](https://github.com/user-attachments/assets/c1fc4e77-14cc-4726-9faf-4ddc78f4d762) |
| 表面 promicro部 | ![IMG_4317](https://github.com/user-attachments/assets/3a4dd613-a674-4855-b304-1c9244438e1f) |

**Comment by Yowkees** [comment, marked as answer]
詳細なご説明ありがとうございます。

一番右側のジャンパに失敗していると認識しました。
下記画像の緑点２点をエナメル線かゴム被覆導線でショートしていただくと、黒くなってしまったジャンパをショートしたのと同じ回路になり、OLED動作すると思います。

![KB39_OLEDfix](https://github.com/user-attachments/assets/57ba1604-c9e2-428b-87a0-619abbdb9c4e)

お手数ですがご対応お願いいたします。

**Reply by Shuyawa89** [reply]
早急なご対応ありがとうございます。

ご教授いただいた箇所をショートさせたところ正常にOLEDの方が点灯しました！
お忙しい中対応していただきありがとうございました🙇

**Comment by Yowkees** [comment]
無事に点灯したようで良かったです！安心しました。
ぜひ末永くご愛用いただければ幸いです。
